lib/jasmine-2.3.4/boot.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
don't commit all your vendor files to git. Use a vendor directory or similar, and place it in your .gitignore.